osu.Game.Tests/Visual/Online/TestSceneUserPanel.cs

```diff
@@ -38,6 +38,9 @@ public class TestSceneUserPanel : OsuTestScene
         {
             UserGridPanel flyte;
 
+            activity.Value = null;
+            status.Value = null;
```

Comment:
Would have probably taken off the `readonly` modifier and reconstructed every test but this works too I suppose.